pkgs/misc/game-rs/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, rustPlatform
+, fetchurl
+, steam-run
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "game-rs";
+  version = "0.1.1-1";
+
+  src = fetchurl {
```

Comment:
This part

Reply:
i was using that previously but it gave "cannot unpack this archive" error

Reply:
Did you ever figure this out?

Reply:
it's weird really, i tried someone else's release archive and it worked but not on this specific one. Maybe changing the version to a use a different archive could work? i  could try that

Reply:
Yup, it is working on archive of a different version, updating title of PR as well